counter-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: ticktock
```

Comment:
We may want to default to applicationName.channelName at SCS level (noting that applicationName is empty, by default) and then skip this property entirely in the modules themselves (with the understanding that the properties are overridable, and document that in the README). That, considering that a PR is the worst place for design discussion :)